apps/e2e-test/README.md

```diff
@@ -0,0 +1,114 @@
+# E2E-Test Framework
+
+The test automation framework for end-to-end testing with Cucumber-JS and Playwright. It abstracts browser interaction to simple steps that are written from a user perspective.
+
+## Prerequisites
+
+In order to run the automated test, you will need the following tools installed on your machine.
+
+- Node v16.16.0
+- Yarn v1.22.19
+
+## Project structure
+
+This project is setup as a yarn based nx-managed monorepo, and e2e tests are located under apps/e2e-test/\* and contains the following:
+
+- features: contains the bdd feature files which has scenarios
+- selector-mappings: contains the element locators stored in yaml or json file with component and page model
+- src: contains the config, hooks, logger, utils, step definitions, reporter and playwright tool specific actions
+- scripts: contains the bash script to install publisher and viewer app locally
+- reports: contains the test execution artifacts and HTML report
+- playwright.config.ts: contains the test config
+
+## Installation and setup
+
+To install the necessary libraries, clone the repo. Then from the root directory of the project, run the following command in a terminal window:
+
+```bash
+yarn
+```
+
+## Publisher and Viewer app installation locally [Optional]
```

Comment:
```suggestion
## [Optional] Publisher and Viewer app installation locally 
```

Reply:
Addressed the review comment: Updated to '[Optional] Publisher and Viewer app installation locally'